ls1intum/Artemis
# `General`: Remove 'experimental' flag and feature advertisement from dark mode popover

### Checklist
#### General


#### Client

### Motivation and Context



- The Dark Mode popover marks the dark mode as 'experimental' at the moment. However, after several fixes we consider the dark mode to be in a stable state, so we'd like to remove it.
- The dark mode currently opens automatically on first client launch to advertise the dark mode to the users. However, after several months all current users should now be aware of this - therefore, the popover should no longer do this.

### Description


- Removed 'experimental' mark
- Removed 'try the dark mode now' stuff and auto-open on first client launch
- Removed unused code that was only there for these features + adapted the test

### Steps for Testing

Prerequisites:
- 1 User

1. Run the client in incognito mode. Verify that the theme is still applied based on OS settings, but the popover does not show up automatically.
2. Verify that the popover still appears if you hover over the theme switcher in the navbar
3. Verify that the experimental flag and all stuff below the OS setting switch is gone
4. Verify that the popover disappears after 250ms after you move the mouse out of it
5. Verify that the popover closes immediately if you click outside of it

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


| Class/File | Branch | Line |
|------------|-------:|-----:|
| navbar.component.ts  | 82% | 92% |

### Screenshots


![grafik](https://user-images.githubusercontent.com/23171488/189374658-5e790d2b-f506-45bc-9a34-0b5d8b65503c.png)
